src/Sarif/Core/SarifLog.cs

```diff
@@ -51,18 +51,24 @@ public static SarifLog Load(string sarifFilePath)
         /// <returns>SarifLog instance for file</returns>
         public static SarifLog Load(Stream source, bool deferred = false)
         {
-            JsonSerializer serializer = new JsonSerializer();
+            var serializer = new JsonSerializer();
 
             if (deferred)
             {
                 serializer.ContractResolver = new SarifDeferredContractResolver();
+                var jptr = new JsonPositionedTextReader(Stream.Synchronized(source));
+                return serializer.Deserialize<SarifLog>(jptr);
             }
-
-            using (StreamReader sr = new StreamReader(source))
-            using (JsonTextReader jtr = new JsonTextReader(sr))
+            else
             {
-                // NOTE: Load with JsonSerializer.Deserialize, not JsonConvert.DeserializeObject, to avoid a string of the whole file in memory.
-                return serializer.Deserialize<SarifLog>(jtr);
+                using (var sr = new StreamReader(source))
```

Comment:
>sr [](http://example.com/codeflow?start=27&length=2)

we don't use abbreviations like 'sr' and 'jtr'. Please use full words, like 'reader', 'streamReader' or 'jsonTestReader

Reply:
This code wasn't changed, just indented.

https://github.com/microsoft/sarif-sdk/blob/45e21b5671100f48807fb74e085342f196ce7513/src/Sarif/Core/SarifLog.cs#L61-L66

Reply:
Gabe just renamed.